### ISSUE TITLE
Accessibility : Lang attribute is missing

### ISSUE THREAD

**Issue by z15bdesc** [Q-V-91/1sttry#2]
https://github.com/Q-V-91/1sttry/blob/f169d1b8cb607ae0baeee73c09afb84f8286e5f7/home.html#L3

You may increase the accessibility of your page through specifying the lang attribute of html markup as detailed in the Mozilla documentation : https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang
